services/dis-identity-operator/.devcontainer/post-install.sh

```
@@ -1,16 +1,16 @@
 #!/bin/bash
 set -x
 
-curl -Lo ./kind https://kind.sigs.k8s.io/dl/latest/kind-linux-amd64
+curl -Lo ./kind https://kind.sigs.k8s.io/dl/latest/kind-linux-$(go env GOARCH)
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)